docs/UserGuide.md

```diff
@@ -51,98 +51,141 @@ Paidlancers is a **desktop app for keeping track of your freelancing events, opt
 
 ## Features
 
-### Add a Contact
-Adding a contact : `newcontact`
+### Adding a contact : `newcontact`
+
 Adding a contact to contacts
-Format: newcontact NAME NUMBER
-Creates a new contact with specified NAME and NUMBER
-Example:
-newcontact Deborah Tan /num 91234567
 
+**Format** : `newcontact n/NAME p/NUMBER`
```

Comment:
Minor issue of extra spacing before ":" here.

Reply:
👍